spec/05-classes-and-objects.md

```diff
@@ -502,10 +502,11 @@ definition apply to all constituent definitions.  The rules governing
 the validity and meaning of a modifier are as follows.
 
 ### `private`
-The `private` modifier can be used with any definition or
-declaration in a template.  Such members can be accessed only from
+The `private` modifier can be used with any template definition or any definition or
```

Comment:
"any template" is not accurate, local definitions can't be `private`. Maybe also say "top-level" here?

```
scala> def f = { private class C; 0 }
                 ^
       error: illegal start of statement (no modifiers allowed here)
```

Reply:
Maybe split it up then?

```
The `private` modifier can be used with any definition or declaration in a
template. Private members of a template can be accessed only from within the
directly enclosing template and its companion module or
[companion class](#object-definitions).

The `private` modifier is also valid for
[top-level](09-top-level-definitions.html#packagings) templates. Private
top-level templates can be accessed only from within the same package.
```

Reply:
Yes I think that makes sense. I'd mark it as "Note" and only keep ``The `private` modifier is also valid for
[top-level](09-top-level-definitions.html#packagings) templates`` instead of repeating it here.